build/AudioToolbox/lib/AudioToolboxLib.vcxproj

```diff
@@ -33,10 +33,15 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\MusicSequence.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\MusicTrack.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\SystemSound.mm" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\WinPhone.cpp">
```

Comment:
If you absolutely need this file, please choose a more descriptive name. WinPhone is incredibly broad and doesn't seem appropriate for a vibration helper function.